data/translations/hi/state-data.json

```diff
@@ -3,310 +3,410 @@
     "hp_link": "https://www.sos.alabama.gov/alabama-votes",
     "registration_link": "https://www.alabamainteractive.org/sos/voter_registration/voterRegistrationWelcome.action",
     "more_info_link": "https://www.sos.alabama.gov/alabama-votes/voter/register-to-vote",
-    "ip_deadline": "मतदान दिवस से 15 दिन पहले ",
-    "online_deadline": "मतदान दिवस से 15 दिन पहले ",
-    "mail_postmarked_deadline": "मतदान दिवस से 15 दिन पहले पोस्टमार्क होना चाहिए",
+    "ip_deadline": "सोमवार, 24 अक्तूबर/अक्टूबर 2022",
+    "online_deadline": "सोमवार, 24 अक्तूबर/अक्टूबर 2022",
+    "mail_postmarked_deadline": "(सोमवार, 24 अक्तूबर/अक्टूबर 2022) तक पोस्टमार्क होना चाहिए",
```

Comment:
We need to remove the `()` from postmarked and received deadlines in this file. We only had the translators do this so we can identify the start and end of the date in case we need to modify it later.